engine/gameobject/src/dmsdk/gameobject/gameobject.h

```diff
@@ -664,6 +664,15 @@ namespace dmGameObject
      */
     void SetScale(HInstance instance, dmVMath::Vector3 scale);
 
+    /*# set scale only for width and height (x and y)
```

Comment:
Are we talking about scale x and y as width and height elsewhere? I don't remember that we do. Perhaps better to just keep referring to it as "x and y scale" or "x and y axis" or perhaps only as "x and y"?

Reply:
Thank you. Fixed!